lib/matplotlib/bezier.py

```diff
@@ -279,7 +278,7 @@ def split_path_inout(path, inside, tolerence=0.01, reorder_inout=False):
         codes_left = [Path.CURVE4, Path.CURVE4, Path.CURVE4]
         codes_right = [Path.MOVETO, Path.CURVE4, Path.CURVE4, Path.CURVE4]
     else:
-        raise ValueError()
+        raise ValueError
```

Comment:
Instead of returning the class, it would be better to supply *some* sort of error message, wouldn't it?  Leaving the empty parenthesis is at least a flag for someone to supply that message.

Reply:
Edited accordingly.  Given the context of the code I believe this line should never be reached (it is effectively following a switch on the result of an internal routine).